llama_stack/apis/agents/openai_responses.py

```diff
@@ -126,6 +127,32 @@ class OpenAIResponseOutputMessageMCPListTools(BaseModel):
 register_schema(OpenAIResponseOutput, name="OpenAIResponseOutput")
 
 
+# This has to be a TypedDict because we need a "schema" field and our strong
+# typing code in the schema generator doesn't support Pydantic aliases. That also
+# means we can't use a discriminator field here, because TypedDicts don't support
+# default values which the strong typing code requires for discriminators.
+class OpenAIResponseTextFormat(TypedDict, total=False):
+    """Configuration for Responses API text format.
+
+    :param type: Must be "text", "json_schema", or "json_object" to identify the format type
+    :param name: The name of the response format. Only used for json_schema.
+    :param schema: The JSON schema the response should conform to. In a Python SDK, this is often a `pydantic` model. Only used for json_schema.
+    :param description: (Optional) A description of the response format. Only used for json_schema.
+    :param strict: (Optional) Whether to strictly enforce the JSON schema. If true, the response must match the schema exactly. Only used for json_schema.
+    """
+
+    type: str
```

Comment:
can this be `Literal["text"] | Literal["json_schema"] | Literal["json_object"]` ?

Reply:
Yes, it can! I was tired when I wrote this and remember trying `Literal["text", "json_schema", "json_object"]` and that did not work but I realize now that was just my fault. I've updated this to constrain the valid values with the literals.